docs/cristin-proxy-swagger.yaml

```diff
@@ -5248,6 +5309,17 @@ components:
         identifier: '1197'
         labels:
           en: 'Educational management'
+    PersonsOrcidExample:
+      value:
+        '@context': 'https://example.org/persons-orcid-context.json'
+        type: 'PersonsOrcid'
+        id: 'https://api.dev.nva.aws.unit.no/cristin/person/orcid'
+        size: 2
+        hits:
+          - id: 'https://api.dev.nva.aws.unit.no/cristin/person/456738'
+            orcid: 'https://api.dev.nva.aws.unit.no/cristin/person/1234-1234-1234-1234'
```

Comment:
ORCID har format "https://orcid.org/0000-0002-1825-0097", men her tenkes det at vi returnerer API-URI?

Reply:
Her er det vårt eget API ja, men burde vi returnere orcid sin egen i tillegg? Hva tenker du? Og hva kaller vi isåfall feltnavnene?

Reply:
Kanskje du kunne kommet med et eksempel på hvordan du vil ha det, så kan jeg endre. Vi støtter ihvertfall oppslag i vårt api på orcid, så id'ene er gyldige

Reply:
Har du forslag til datamodell @brinxmat ? Jeg må vite om den må endres før jeg merger...